### DISCUSSION TITLE
⭐  Star Exchange 🌟

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [drknzz/GitHub-Achievements#119] [General]
Star my repo - https://github.com/Sriansh-raj/GitHub-Achievements

and I will star yours back 😃 

(reply with your repo link)

**#starstruck** **#starstruck-badge** ❤️

**Comment by Rantoryu** [comment]
@Sriansh-raj Hey, I've starred your repo, here's mine :) - https://github.com/Rantoryu/Personal_Website
#20 #40

**Reply by Sriansh-raj** [reply]
Done 😄

**Reply by laurentiucozma12** [reply]
I gave you a star, can you give me a star please [Epic Game News](https://github.com/laurentiucozma12/epicgamenews-presentation) ?

**Reply by Rantoryu** [reply]
@laurentiucozma12 done :)

**Reply by laurentiucozma12** [reply]
thank you @Rantoryu

**Reply by shreya-oeo** [reply]
Done 
Here's mine https://github.com/aulolua/chatroom

**Reply by Guexss** [reply]
Done,  https://github.com/mmanosalva/RSA-Mathematica

**Reply by luisfb** [reply]
done.
here is mine:
https://github.com/luisfb/asuna-full-text-search

**Reply by NobleMajo** [reply]
nice i will star, here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Reply by luisfb** [reply]
@NobleMajo done

**Reply by KamBRJC** [reply]
star all yours repos guys 

please star mine https://github.com/z-nnovation/GitSwitcher

**Reply by jvcByte** [reply]
@Rantoryu @Guexss @luisfb @NobleMajo All done, mine here: https://github.com/jvc-byte/Simple-Bank-Using-solidity.
@KamBRJC did yours before.

**Comment by laurentiucozma12** [comment]
I gave you a star, can you give me a star please [Epic Game News](https://github.com/laurentiucozma12/epicgamenews-presentation) ?

**Reply by Sriansh-raj** [reply]
@laurentiucozma12 👍🏻done :D

**Reply by laurentiucozma12** [reply]
thank you all

**Reply by shreya-oeo** [reply]
Hey im here too https://github.com/aulolua/chatroom

**Reply by NobleMajo** [reply]
i like the game news idea, here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Reply by FlutterSmith** [reply]
https://github.com/FlutterSmith/FlutterSmith

**Reply by KamBRJC** [reply]
@FlutterSmith star your repo, please give back https://github.com/z-nnovation/GitSwitcher

**Reply by jvcByte** [reply]
@aulolua @FlutterSmith done, mine here: https://github.com/jvc-byte/Simple-Bank-Using-solidity.

@laurentiucozma12  i couldn't locate your repo... please share again

**Reply by ganochenkodg** [reply]
@felizardo27 @jvc-byte @FlutterSmith done, and mine pls https://github.com/ganochenkodg/qdrant-operator

**Comment by pvhuwung** [comment]


**Reply by Sriansh-raj** [reply]
👍🏻 @pvhuwung I've starred yours

**Reply by shreya-oeo** [reply]
Count me in too https://github.com/aulolua/chatroom

**Reply by NobleMajo** [reply]
@pvhuwung what project? all? here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Comment by shreya-oeo** [comment]
https://github.com/aulolua/chatroom
Starred yours 👍.

**Reply by Sriansh-raj** [reply]
@aulolua done :D

**Comment by sheeehy** [comment]
https://github.com/sheeehy/Geo-Git-v1  <3

**Reply by NobleMajo** [reply]
<3 geo, here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Reply by FlutterSmith** [reply]
https://github.com/FlutterSmith/FlutterSmith

**Reply by KamBRJC** [reply]
star back https://github.com/z-nnovation/GitSwitcher

**Comment by ganochenkodg** [comment]
@Sriansh-raj @Rantoryu @aulolua @sheeehy  done, star mine pls https://github.com/ganochenkodg/qdrant-operator

**Reply by shreya-oeo** [reply]
Okay do mine too

**Reply by luisfb** [reply]
done.
here is mine:
https://github.com/luisfb/asuna-full-text-search

**Reply by NobleMajo** [reply]
@ganochenkodg done :D, here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Reply by KamBRJC** [reply]
hi, star back bro https://github.com/z-nnovation/GitSwitcher

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by luisfb** [reply]
done.
here is mine:
https://github.com/luisfb/asuna-full-text-search

**Reply by NobleMajo** [reply]
also done :P here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Reply by luisfb** [reply]
done

**Reply by FlutterSmith** [reply]
https://github.com/FlutterSmith/FlutterSmith done

**Comment by Salgado2004** [comment]
That's a great idea
Can I join? [https://github.com/Salgado2004/Angular-Ecommerce-UFPR-Faz-Sistema](https://github.com/Salgado2004/Angular-Ecommerce-UFPR-Faz-Sistema)

Gonna star your repos back

**Reply by luisfb** [reply]
Done.
here is mine:
https://github.com/luisfb/asuna-full-text-search

**Reply by KamBRJC** [reply]
done, star mine https://github.com/z-nnovation/GitSwitcher

**Comment by rehan509** [comment]
To get the Galaxy Brain achievement you need to:

answer at least 2 discussions and
have the OP mark your answers to the questions.
Hope that helps and you get Galaxy Brain soon ;)

**Comment by luisfb** [comment]
done.
Here is mine:
https://github.com/luisfb/asuna-full-text-search

**Reply by NobleMajo** [reply]
done :D here are mine:
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Comment by NobleMajo** [comment]
i like the idea :D lets share stars: 
go encryption cli tool: https://github.com/NobleMajo/vault
typescript ssh library: https://github.com/noblemajo/hivessh

**Comment by larry-xue** [comment]
done
here is mine: https://github.com/larry-xue/larry-xue-site

**Comment by Denisyasyuchenya** [comment]
give you a star

star my https://github.com/Denisyasyuchenya/my-location-app

**Comment by KamBRJC** [comment]
give me a star please https://github.com/z-nnovation/GitSwitcher

**Comment by jvcByte** [comment]
> Star my repo - https://github.com/Sriansh-raj/GitHub-Achievements
> 
> and I will star yours back 😃
> 
> (reply with your repo link)
> 
> **#starstruck** **#starstruck-badge** ❤️

@Sriansh-raj done. mine: https://github.com/jvc-byte/Simple-Bank-Using-solidity

**Reply by Sriansh-raj** [reply]
@jvc-byte Done bro 👍🏻

**Reply by jvcByte** [reply]
@Sriansh-raj thanks man
